docs/supported.md

```diff
@@ -361,7 +361,8 @@ Direct Input: $+ - / * ⋅ ± × ÷ ∓ ∔ ∧ ∨ ∩ ∪ ≀ ⊎ ⊓ ⊔ ⊕
 | $\cosec$ `\cosec`  | $\hom$ `\hom`  | $\tg$ `\tg`  | $\Pr$ `\Pr`  |
 | $\cosh$ `\cosh`| $\ker$ `\ker`  | $\th$ `\th`  | $\sup$ `\sup` |
 | $\cot$ `\cot` | $\lg$ `\lg`| $\operatorname{f}$ `\operatorname{f}` | $\argmax$ `\argmax` |
-|$\argmin$ `\argmin`|
+| | | |$\argmin$ `\argmin`|
+| | | | $\plim$ `\plim` |
```

Comment:
I think it's fine for `argmin` to appear first.  If you want organize this table better a separate PR would be preferable.